src/coreclr/jit/lower.cpp

```diff
@@ -4460,6 +4463,18 @@ bool Lowering::TryLowerConditionToFlagsNode(GenTree* parent, GenTree* condition,
         }
 #endif
 
+#if !defined(TARGET_LOONGARCH64) && !defined(TARGET_RISCV64)
+        if (!allowMultipleFlagsChecks && cond->IsFloat())
```

Comment:
```suggestion
        if (!allowMultipleFlagsChecks)
```

Nit (the check below also doesn't have this)